scripts/zones/RuAun_Gardens/mobs/Suzaku.lua

```diff
@@ -36,5 +46,30 @@ function onMonsterMagicPrepare(mob, target)
     else
         return 235; -- burn
     end
+end;
 
+-----------------------------------
+-- onAdditionalEffect
+-----------------------------------
+
+function onAdditionalEffect(mob, target, damage)
+    local levelDiff = target:getMainLvl() - mob:getMainLvl();
+    local statDiff = mob:getStat(MOD_INT) - target:getStat(MOD_INT);
+    
+    local dmg = statDiff + levelDiff + math.random(0, 15); -- INT modifier + difference in level + variance
+    local params = {};
+    params.bonusmab = 0;
+    params.includemab = false;
+    
+    dmg = addBonusesAbility(mob, ELE_FIRE, target, dmg, params);
+    dmg = dmg * applyResistanceAddEffect(mob, target, ELE_FIRE, 0);
+    dmg = adjustForTarget(target, dmg, ELE_FIRE);
+    
+    if (dmg < 0) then

```

Comment:
If less than zero, change the message id (with the right gear, you can in fact get healed by the "damage").